File: Python3/ITP1_05_C.py
while True:
    n = [int(i) for i in input().split()]
    if n[0] == 0 and n[1] == 0:
        break
    for i in range(0, n[0]):
        for j in range(0, n[1]):
            if (i + j) % 2 == 0:
                print("#", end="")
            else:
                print(".", end="")
        print()
    print()

File: Ruby/ITP1_05_C.rb
while true
  a = gets.split(" ").map(&:to_i)
  if a[0] == 0 && a[1] == 0 then
    break
  end
  1.upto(a[0]){|i|
    1.upto(a[1]){|j|
      if (i + j) % 2 == 0 then
        print "#"
      else
        print "."
      end
    }
    puts ""
  }
  puts ""
end
